src/mplutils.h

```diff
@@ -46,4 +46,19 @@ enum {
 
 const size_t NUM_VERTICES[] = { 1, 1, 1, 2, 3, 1 };
 
+inline int prepare_and_add_type(PyTypeObject *type, PyObject *module)
+{
+    if (PyType_Ready(type)) {
+        return -1;
+    }
+    char const* ptr = strrchr(type->tp_name, '.');
+    if (!ptr) {
+        return -1;
+    }
+    if (PyModule_AddObject(module, ptr + 1, (PyObject *)type)) {
+        return -1;
```

Comment:
Should we be worried that the two other error cases _did_ get covered?

Reply:
I agree this seems... rather strange?  Given that everything imports fine...